docs/compilation/compile_models.rst

```diff
@@ -5,18 +5,51 @@ This page describes how to compile a model with MLC LLM. Model compilation takes
 and optimized model lib for a given platform. It enables users to bring their own new model weights, try different quantization modes,
 and customize the overall model optimization flow.
 
-
 .. note::
     Before you proceed, please make sure that you have :ref:`install-tvm-unity` correctly installed on your machine.
     TVM-Unity is the necessary foundation for us to compile models with MLC LLM.
     If you want to build webgpu, please also complete :ref:`install-web-build`.
     Please also follow the instruction in :ref:`deploy-cli` to obtain the CLI app that can be used to chat with the compiled model.
     Finally, we strongly recommend you read :ref:`project-overview` first to get familiarized with the high-level terminologies.
 
+
 .. contents:: Table of Contents
     :depth: 1
     :local:
 
+Install MLC-LLM Package
+-----------------------
+
+Option 1: Work with Source Code
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The eaisest way is to use MLC-LLM is to clone the repostiory, and compile models under the root directory of the repository.
+
+.. code:: bash
+
+    # clone the repository
+    git clone git@github.com:mlc-ai/mlc-llm.git --recursive
+    # enter to root directory of the repo
+    cd mlc-llm  
+
+Option 2: Use Prebuilt Package
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you need to compile models outside the MLC-LLM codebase, you can install the prebuilt pip wheels of MLC-LLM package by
+following the instructions in `<https://mlc.ai/package/>`_.
+
```

Comment:
Is it better to add this option after the package successfully set up? I wonder if it is too early to provide this option here.

Reply:
I mean we first get this PR in, build the package and verify that it is all set, and then come back to update the doc with prebuilt package.

Reply:
Yeah I agree with you.